### PR TITLE
summary table update

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*https://github.com/bcgov/entity/issues/18664

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-66-sstq5c0x.web.app

**Comment by kialj876** [comment]
forgot to add internationalization -- updating now

File: btr-web/btr-main-app/pages/myRegistriesDetails.vue [hunk 1]
```diff
@@ -25,13 +25,7 @@
...
           </template>
           <template #info-address="{ row }">
-            <!-- FUTURE: common address display component -->
-            <p>{{ row.info.line1 }}</p>
-            <p v-if="row.info.line2">
-              {{ row.info.line2 }}
-            </p>
-            <p>{{ row.info.city }} {{ row.info.region }}&nbsp;&nbsp;{{ row.info.postalCode }}</p>
-            <p>{{ row.info.country.name }}</p>
+            <BcrosInputsAddressDisplay :model-value="row.info" />
```
Comment (hfekete) [inline btr-web/btr-main-app/pages/myRegistriesDetails.vue:28]: Cool! Updating when see comments.

**Review by hfekete — COMMENTED** [review]
Looks good other then i18n, ping me when you have the texts added !

File: btr-web/btr-main-app/pages/myRegistriesDetails.vue (hunk 1 above)
Comment (kialj876) [reply btr-web/btr-main-app/pages/myRegistriesDetails.vue:28]: yeah I needed the same display in the table so I figured it was a good time to add that

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-66-sstq5c0x.web.app

File: btr-web/btr-main-app/components/individual-person/AddNew.vue [hunk 2]
```diff
@@ -242,7 +242,8 @@ const defaultSI = {
     sharesVotes: {
       registeredOwner: false,
       beneficialOwner: false,
-      indirectControl: false
+      indirectControl: false,
+      inConcertControl: true
```
Comment (hfekete) [inline btr-web/btr-main-app/components/individual-person/AddNew.vue:246]: do we want this one to be true by default ?

File: btr-web/btr-main-app/components/individual-person/SummaryTable.vue [hunk 3]
```diff
@@ -3,53 +3,124 @@
...
-        <div class="px-6 py-3 align-top min-w-[120px]">
-          {{ row.details?.dateOfBirth }}<br>
-          <span v-for="(residency, index) in row.details?.residency" :key="index">{{ residency }}<br></span>
-          <!-- todo: internationalize tax resident -->
-          <span v-if="row.details?.isTaxResident">Tax Resident</span>
+        <div data-cy="summary-table-details">
+          <span>{{ displayDate(row.profile.birthDate) }}</span><br>
+          <span v-if="row.profile.taxNumber">{{ row.profile.taxNumber }}<br></span>
+          <label>{{ $t('labels.citizenships') }}:</label><br>
+          <span v-if="row.profile.citizenshipCA !== 'other'">Canada<br></span>
```
Comment (hfekete) [inline btr-web/btr-main-app/components/individual-person/SummaryTable.vue:25]: Was this label left out on purpose ?

File: btr-web/btr-main-app/components/individual-person/SummaryTable.vue [hunk 4]
```diff
@@ -3,53 +3,124 @@
...
+}
+
+function getControlTextField (items: { value: boolean, label: string }[]) {
+  const activeLabels = items.filter(item => item.value).map(item => item.label)
+  return activeLabels.join('') || ''
+}
+
+function getSharesControlText (significantIndividual: SignificantIndividualI) {
+  const field = getControlTextField([
+    { value: significantIndividual.controlType.sharesVotes.registeredOwner, label: 'registered' },
```
Comment (hfekete) [inline btr-web/btr-main-app/components/individual-person/SummaryTable.vue:98]: missed those

File: btr-web/btr-main-app/components/individual-person/SummaryTable.vue [hunk 5]
```diff
@@ -3,53 +3,124 @@
...
+        sharePercent: significantIndividual.percentOfShares || '0',
+        votePercent: significantIndividual.percentOfVotes || '0'
+      })
+  }
+  return ''
+}
+
+function getDirectorsControlText (directorsConstrol: ControlOfDirectorsI) {
+  const field = getControlTextField([
+    { value: directorsConstrol.directControl, label: 'direct' },
```
Comment (hfekete) [inline btr-web/btr-main-app/components/individual-person/SummaryTable.vue:115]: missed those

File: btr-web/btr-main-app/components/individual-person/AddNew.vue (hunk 2 above)
Comment (kialj876) [reply btr-web/btr-main-app/components/individual-person/AddNew.vue:246]: Nope, nice catch. Just put that in to test the text showing up

File: btr-web/btr-main-app/components/individual-person/SummaryTable.vue (hunk 3 above)
Comment (kialj876) [reply btr-web/btr-main-app/components/individual-person/SummaryTable.vue:25]: missed it :P

**Pull request merged**